seatunnel-engine/seatunnel-engine-core/src/main/java/org/apache/seatunnel/engine/core/parse/MultipleTableJobConfigParser.java

```diff
@@ -172,7 +181,14 @@ public MultipleTableJobConfigParser(
         this.jobConfig = jobConfig;
         this.commonPluginJars = commonPluginJars;
         this.isStartWithSavePoint = isStartWithSavePoint;
-        this.seaTunnelJobConfig = ConfigBuilder.of(Paths.get(jobDefineFilePath), variables);
+        boolean metalakeEnabled =
+                Boolean.parseBoolean(System.getenv().getOrDefault("METALAKE_ENABLED", "false"));
+        if (metalakeEnabled) {
+            this.seaTunnelJobConfig =
+                    getMetalakeConfig(ConfigBuilder.of(Paths.get(jobDefineFilePath), variables));
+        } else {
+            this.seaTunnelJobConfig = ConfigBuilder.of(Paths.get(jobDefineFilePath), variables);
+        }
```

Comment:
We need to ensure that Flink/Spark/Zeta all support this feature simultaneously.

Therefore, it needs to be adapted in both the `FlinkTaskExecuteCommand`and `SparkTaskExecuteCommand` classes.